Essentials/src/com/earth2me/essentials/signs/SignSell.java

```diff
@@ -18,13 +20,21 @@ public SignSell() {
 
     @Override
     protected boolean onSignCreate(final ISign sign, final User player, final String username, final IEssentials ess) throws SignException {
-        validateTrade(sign, 1, 2, player, ess);
-        validateTrade(sign, 3, ess);
+        validateTrade(sign, 1, 2, player, ess);  
+        if(sign.getLine(3).isEmpty()) {
+            updateFromWorth(sign, player, ess, new BigDecimal("1.0"));
+        }
+        else {
```

Comment:
```suggestion
        if (sign.getLine(3).isEmpty()) {
            updateFromWorth(sign, player, ess, new BigDecimal("1.0"));
        } else {
```

This is also common functionality between buy and sell which ought to be in a separate class.